test/integration/resize_test.go

```diff
@@ -22,29 +22,29 @@ var _ = Describe("vary VM parameters: memory cpus, disk", Label("openshift-prese
 		It("start CRC", func() {
 			// default values: "--memory", "9216", "--cpus", "4", "disk-size", "31"
 			if bundlePath == "" {
-				Expect(RunCRCExpectSuccess("start", "-p", pullSecretPath)).To(ContainSubstring("Started the OpenShift cluster"))
+				Expect(RunCRCExpectSuccess("start", "--memory", "12000", "--cpus", "5", "--disk-size", "40", "-p", pullSecretPath)).To(ContainSubstring("Started the OpenShift cluster"))
 			} else {
-				Expect(RunCRCExpectSuccess("start", "-b", bundlePath, "-p", pullSecretPath)).To(ContainSubstring("Started the OpenShift cluster"))
+				Expect(RunCRCExpectSuccess("start", "--memory", "12000", "--cpus", "5", "--disk-size", "40", "-b", bundlePath, "-p", pullSecretPath)).To(ContainSubstring("Started the OpenShift cluster"))
 			}
 		})
 
 		It("check VM's memory size", func() {
 			out, err := SendCommandToVM("cat /proc/meminfo")
 			Expect(err).NotTo(HaveOccurred())
-			Expect(out).Should(MatchRegexp(`MemTotal:[\s]*9\d{6}`))
+			Expect(out).Should(MatchRegexp(`MemTotal:[\s]*11\d{6}`))
 		})
 
 		It("check VM's number of cpus", func() {
 			out, err := SendCommandToVM("cat /proc/cpuinfo")
```

Comment:
not a blocker but just a suggestion , why are we not using `lscpu` and then use `CPU\(s\):[\s]*5` which provide us correct cpus instead one less because `/proc/cpuinfo` start it from `0`? May be another PR to try and fix it?